manta-benchmark/Cargo.toml

```diff
@@ -33,8 +33,14 @@ harness = false
 
 [dependencies]
 manta-accounting = { path = "../manta-accounting", default-features = false, features = ["test"] }
-manta-crypto = { path = "../manta-crypto", default-features = false, features = ["test"] }
+manta-crypto = { path = "../manta-crypto", default-features = false, features = ["getrandom", "test"] }
 manta-pay = { path = "../manta-pay", default-features = false, features = ["groth16", "test"] }
+getrandom = { version = "0.2", features = ["js"]}
+wasm-bindgen = "0.2"
+web-sys = { version = "0.3", features = ["console"]}
 
 [dev-dependencies]
 criterion = { version = "0.3.4", default-features = false }
+
+[lib]
+crate-type = ["cdylib", "lib"]
```

Comment:
Move this to above `[[bench]]`. The order is `[lib]`, `[[bin]]`, then `[[bench]]`.